Sources/RSDKUtilsMain/Extensions/UIColor+Extensions.swift

```diff
@@ -1,3 +1,4 @@
+#if os(iOS)
```

Comment:
You said that UIColor was available on watchOS some maybe let's add  `|| os(watchOS)` here 🙂